cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -236,6 +243,14 @@ func (k *Kuberhealthy) Start(ctx context.Context) {
 				k.RestartChecks(ctx)
 				k.RestartReaper(ctx)
 			}
+		case <-configReloadChan:
+			log.Infoln("control: Witnessed a kuberhealthy configuration change...")
+
+			// if we are master, stop, reconfigure our khchecks, and start again with the new configuration
+			if isMaster {
+				log.Infoln("control: Reloading external check configurations due to kuberhealthy configuration update")
+				k.RestartChecks(ctx)
```

Comment:
@integrii should we also be restarting the Reaper -- like in the case above? 
https://github.com/kuberhealthy/kuberhealthy/pull/942/files#diff-6fe2eb1bda3003e83c281ea43553e293915389e31314d523116afe24efe877e3R244

Reply:
I would restart the reaper just in case.  This way we are more likely to make any new configuration options that have been set for it take effect.